scripts/import_cldr.py

```diff
@@ -480,6 +483,10 @@ def parse_locale_display_names(data, tree):
         for listPattern in listType.findall('listPatternPart'):
             list_patterns[listPattern.attrib['type']] = _text(listPattern)
 
+    measurement_systems = data.setdefault('measurement_systems', {})
+    for measurement_system in tree.findall('.//measurementSystemNames/measurementSystemName'):
+        measurement_systems[measurement_system.attrib['type']] = _text(measurement_system)
```

Comment:
::squints:: I think you can use _import_type_text(measurement_systems, measurement_system).